quarkus/container/Dockerfile

```diff
@@ -18,11 +18,13 @@ RUN mv /tmp/keycloak/keycloak-* /opt/keycloak && mkdir -p /opt/keycloak/data
 RUN chmod -R g+rwX /opt/keycloak
 
 FROM registry.access.redhat.com/ubi8-minimal
+ENV LANG en_US.UTF-8
 
 COPY --from=build-env --chown=1000:0 /opt/keycloak /opt/keycloak
 
 RUN microdnf update -y && \
-    microdnf install -y java-11-openjdk-headless && microdnf clean all && rm -rf /var/cache/yum/* && \
+    microdnf install -y --nodocs java-11-openjdk-headless glibc-langpack-en && microdnf clean all && rm -rf /var/cache/yum/* && \
```

Comment:
Adding `--nodocs` is not strictly related to this PR, so would have been better as a separate PR, but since it's a minor thing I'm okay with it.

Reply:
yeh, see the previous discussion with @jsorah and @vmuzikar - whatever works for you, could also add it as separate PR :)

Reply:
rebased, removed sed and squashed commits. also doublechecked if it works without -> yep. 
![Screenshot 2022-04-22 at 10 38 51](https://user-images.githubusercontent.com/89905860/164651994-8879baea-6afc-45d1-b855-d91d6ccf3563.png)

Reply:
oops, that comment is for the other thread.